src/client/components/sidebar/sidebar.js

```diff
@@ -45,17 +45,16 @@ class Sidebar extends React.Component {
         </Button>
         <hr />
 
-        <Leaderboard passedUsers={this.props.G.passed} playerID={this.props.playerID} scores={this.props.G.scores} names={this.props.names} cards={getDealtCardsForPlayers(this.props.G.order, this.props.G.dealt)} />
+        <Leaderboard gameMode={this.props.G.gameMode} passedUsers={this.props.G.passed} playerID={this.props.playerID} scores={this.props.G.scores} names={this.props.names} cards={getDealtCardsForPlayers(this.props.G.order, this.props.G.dealt)} />
         {isLastToPass && <div className="warning">You are the last one to pass!</div>}
         <Button color={(isLastToPass) ? "warning" : "secondary"} className="pass" size="lg" block disabled={
-
           this.props.ctx.phase !== "threats" ||
           this.props.G.passed.includes(this.props.playerID) ||
           !this.props.active
         } onClick={() => { this.props.moves.pass() }}>
           Pass
         </Button>
-        <DealtCard card={dealtCard} />
+        <DealtCard card={dealtCard} gameMode={currentGameMode} />
```

Comment:
Is there a reason why you cannot directly use `this.props.G.gameMode` at this place?